README.md

```diff
@@ -100,21 +147,6 @@ You can drill down to get more information by given `plot`, `plot_correlation` a
 
 Don't forget to checkout the [examples] folder for detailed demonstration!
 
-### Connector
-
-Connector provides a simple way to collect data from different websites, offering several benefits:
-* A unified API: you can fetch data using one or two lines of code to get data from many websites.
-* Auto Pagination: it automatically does the pagination for you so that you can specify the desired count of the returned results without even considering the count-per-request restriction from the API.
-* Smart API request strategy: it can issue API requests in parallel while respecting the rate limit policy.
-
-In the following examples, you can download the Yelp business search result into a pandas DataFrame, 
-using only two lines of code, without taking deep looking into the Yelp documentation!
-More examples can be found here:
-[Examples](https://github.com/sfu-db/dataprep/tree/develop/examples)
-
-<center><a href="https://sfu-db.github.io/dataprep/connector.html#getting-web-data-with-connector-query"><img src="https://github.com/sfu-db/dataprep/raw/develop/assets/connector.png"/></a></center>
-
-
 ## Contribute
 
 There are many ways to contribute to Dataprep.
```

Comment:
Should this be DataPrep?

Reply:
You're right. I got confused because in the current Readme file in some sections is mentioned "Dataprep" and in others "DataPrep". I'll update the info.

Reply:
Done.